sample-compose/data/src/main/java/co/nimblehq/sample/compose/data/repository/RepositoryImpl.kt

```diff
@@ -4,10 +4,20 @@ import co.nimblehq.sample.compose.data.response.toModels
 import co.nimblehq.sample.compose.data.service.ApiService
 import co.nimblehq.sample.compose.domain.model.Model
 import co.nimblehq.sample.compose.domain.repository.Repository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
 
 class RepositoryImpl constructor(
     private val apiService: ApiService
 ) : Repository {
 
-    override suspend fun getModels(): List<Model> = apiService.getResponses().toModels()
+    override fun getModels(): Flow<List<Model>> = flow {
+        try {
+            val result = apiService.getResponses().toModels()
+            emit(result)
+        } catch (exception: Exception) {
+            // TODO do error mapping here
```

Comment:
Shall we start the TODO comment with Capital?
```suggestion
            // TODO: Do error mapping here
```

Reply:
@lydiasama I will apply the same structure of error mapping in sample projects. Hence it will be removed soon in the [#212] Part 2/2.

Reply:
@luongvo Thanks for your answer. Then this is acceptable.